doc/Language/objects.pod6

```diff
@@ -286,9 +283,9 @@ attributes in initializers.
 
 =head2 Private Methods
 
-Methods that are not going to be called or callable from anywhere outside
+Methods that are not going to be called or are not callable from anywhere outside
```

Comment:
How about:
`Methods with an exclamation mark C<!> before the method
name are not callable from anywhere outside the defining class;
such methods are private in the sense they are not visible
from outside the class that declares them.
Private methods are invoked with an exclamation mark instead of a dot:`